pages/stackers/[sub]/[when].js

```diff
@@ -20,58 +20,58 @@ const WhenComposedChart = dynamic(() => import('@/components/charts').then(mod =
 })
 
 const GROWTH_QUERY = gql`
-  query Growth($when: String!, $from: String, $to: String, $sub: String)
+  query Growth($when: String!, $from: String, $to: String, $sub: String, $subSelect: Boolean = false)
   {
-    registrationGrowth(when: $when, from: $from, to: $to) {
+    registrationGrowth(when: $when, from: $from, to: $to) @skip(if: $subSelect) {
       time
       data {
         name
         value
       }
     }
-    itemGrowth(when: $when, from: $from, to: $to) {
+    itemGrowth(when: $when, from: $from, to: $to) @skip(if: $subSelect) {
       time
       data {
         name
         value
       }
     }
-    spendingGrowth(when: $when, from: $from, to: $to) {
+    spendingGrowth(when: $when, from: $from, to: $to) @skip(if: $subSelect) {
       time
       data {
         name
         value
       }
     }
-    spenderGrowth(when: $when, from: $from, to: $to) {
+    spenderGrowth(when: $when, from: $from, to: $to) @skip(if: $subSelect) {
       time
       data {
         name
         value
       }
     }
-    stackingGrowth(when: $when, from: $from, to: $to) {
+    stackingGrowth(when: $when, from: $from, to: $to) @skip(if: $subSelect) {
       time
       data {
         name
         value
       }
     }
-    stackerGrowth(when: $when, from: $from, to: $to) {
+    stackerGrowth(when: $when, from: $from, to: $to) @skip(if: $subSelect) {
```

Comment:
I used [graphql directives](https://graphql.org/learn/queries/#directives) to avoid fetching data we don't use. Not a big deal but it was straightforward enough for me to add.